source/comtypesMonkeyPatches.py

```diff
@@ -95,14 +96,21 @@ def new__call__(self,*args,**kwargs):
 # This causes Release() to be called more than it should, which is very nasty and will eventually cause us to access pointers which have been freed.
 from comtypes import _compointer_base
 
-_cpbDel = _compointer_base.__del__
+_compointer_base._oldCpbDel = _compointer_base.__del__
 def newCpbDel(self):
+	# __del__ may be called while Python is exiting.
+	# In this state, global symbols may be set to None
+	# Therefore avoid calling into garbageHandler.

```

Comment:
Might be worth adding "or `log`, unless `isFinalizing` is checked first to ensure they are still available"